mailgun/listserv_client.py

```diff
@@ -221,6 +221,14 @@ def send_mail(self, from_address, reply_to_address, to_address, cc_address,
             'html': html
         }
 
+        # we accept a single address or a list of addresses in to_address.
+        # if it's a list, add in recipient_variables to make sure mailgun
+        # doesn't include the whole list in the to: field, per
+        #   https://documentation.mailgun.com/user_manual.html#batch-sending
+        if not isinstance(to_address, basestring):
```

Comment:
We talked offline about `h:cc` and a putative `h:to` and whether we can pass those things through on the mails sent out to the individuals on the list -- leaving that in your hands. :)